setupbase.py

```diff
@@ -129,13 +189,24 @@ def run(self):
                 json.dump(details, f, indent=4, sort_keys=True)
 
             # Dump a .ttf font file:
-            fontPath = self.__get_ttf_path(style)
+            font_path = self.__get_ttf_path(style)
             data = files[style]
-            hashes[style] = hashlib.md5(data).hexdigest()
-            self.__print('Dumping updated "%s" font: %s' % (style, fontPath))
-            with open(fontPath, 'w+') as f:
+            self.__print('Dumping updated "%s" font: %s' % (style, font_path))
+            with open(font_path, 'w+') as f:
                 f.write(data)
 
+            # Fix to prevent repeated font names:
+            if style in ('regular', 'solid'):
+                newName = str("Font Awesome 5 Free %s") % style.title()
```

Comment:
Please change `newName` here to `new_name`.

Reply:
Woops! Where I work our style guide is to use camelCase. I know this project uses snake_case and I made a mental effort to make it so, but I guess habit got the best of me! hehe. I'll edit it.